Helm Charts

netflash opened the pull request:
Reincarnation of #1

netflash committed:
Init helm chart commit

netflash committed:
Temporary add openssh

netflash committed:
Pre-generate ssh-keyfiles

netflash committed:
Revert Dockerfile

netflash committed:
Helm chart - first pass

Commit:
Add search team

netflash committed:
Add helm chart icon

netflash committed:
Empty notes.txt

leoblanc reviewed (approved):
Looks good. And it's nice that we are also using K8s Cron Jobs too! Good work

toddmichael reviewed (approved):
👍

deborahchua committed:
Add Benchmarking team to alphagov config

chao-xian committed:
Update content-tools

@leenagupte and @theleebriggs are leeving 😭. Also beefed up our daily
quotes.

But @cbaines joins!

chao-xian committed:
Add a couple more quotes for Content Tools Team

brenetic committed:
Remove Custom. Booooooooo

binaryberry committed:
Merge pull request #153 from MatMoore/search-team

Add search team

binaryberry committed:
Merge pull request #158 from deborahchua/patch-1

Add Benchmarking team to alphagov config

binaryberry committed:
Merge pull request #161 from brenetic/remove-custom

Remove Custom. Booooooooo

binaryberry committed:
Merge pull request #159 from chao-xian/update-content-tools

Update content-tools

tijmenb committed:
Split Finding Things

We're now content transformation and navigation. Still in the same
channel though!

tijmenb committed:
Merge pull request #164 from binaryberry/split-finding-things

Split Finding Things

tijmenb committed:
Add spec for alphagov.yml

This adds a test for our config. Makes sure that we don't put in
invalid YAML.

binaryberry committed:
Merge pull request #165 from binaryberry/add-test-for-config

Add spec for alphagov.yml

Commit:
Config changes to reflect new teams

binaryberry committed:
Merge pull request #166 from binaryberry/update-config-2017

Config changes to reflect new teams

Commit:
Updated team config

Add members and exclude titles.

stig committed:
Fix typo in README (coma -> comma)

binaryberry committed:
Merge pull request #169 from stig/patch-1

Fix typo in README (coma -> comma)

binaryberry committed:
Merge pull request #167 from theleebriggs/template-consolidation-team-update

Updated template consolidation team

Commit:
Update readme to include configuring bash scripts

Commit:
Update readme to remove references to configuring list of repos

It seems to have been updated with #55

tijmenb committed:
Different channels for taxonomy & nav

We have new channels now.

carvil committed:
Add Andrien to the Navigation team

tijmenb committed:
Merge pull request #172 from carvil/add-andrien-to-nav-team

Add Andrien to the Navigation team

tijmenb committed:
Merge pull request #171 from binaryberry/split-nav-tax

Different channels for taxonomy & nav

tijmenb committed:
Fix taxonomy team name

This prevents `seal me up` from working.

binaryberry committed:
Merge pull request #173 from binaryberry/tijmenb-patch-1

Fix taxonomy team name

binaryberry committed:
Merge pull request #170 from aveline/readme-updates

Readme updates

kevindew committed:
Add hold government to account team

Combination of developers from Content History and Scrutineers, as we'll be
sharing a trello board and channel be good to keep the PRs together.

deanwilson committed:
The team roster has changed slightly.

Remove the people that have changed roles from the govuk-infrastructure
team as we don't need to see their PRs

Commit:
Make the team name match slack for search

The mismatch prevents the "seal me up" command from working.
See https://github.com/binaryberry/seal/pull/173

tijmenb committed:
Merge pull request #175 from deanwilson/update_infra_team

The team roster has changed slightly.

tijmenb committed:
Add taxonomy to the morning seal

tijmenb committed:
Merge pull request #174 from kevindew/hold-gov-to-account

Add hold government to account team

tijmenb committed:
Merge pull request #176 from MatMoore/rename-search-team

Make the team name match slack for search

tijmenb committed:
Merge pull request #177 from binaryberry/taxon-am

Add taxonomy to the morning seal

Commit:
Add Worldwide Publishing team to morning seal

tijmenb committed:
Merge pull request #178 from rubenarakelyan/add-worldwide-publishing

Add Worldwide Publishing team to morning seal

Commit:
Make seal pick up my PRs

The seal was not returning my PRs - I think because the filter
is case sensitive.

tijmenb committed:
Merge pull request #179 from MatMoore/notice-me-seal

Make seal pick up my PRs

Commit:
Remove email team config

binaryberry committed:
Merge pull request #182 from binaryberry/remove-email

Remove email team config
